win32ss/drivers/videoprt/videoprt.c

```diff
@@ -794,6 +798,9 @@ VideoPortDebugPrint(
 {
     va_list ap;
 
+    if (VideoDebugLevel >= DebugPrintLevel)
+        DebugPrintLevel = Error;
```

Comment:
Shouldn't it be something like...
```suggestion
    if (VideoDebugLevel != DebugPrintLevel)
        DebugPrintLevel = VideoDebugLevel;
```
Because in your code `DebugPrintLevel` is always assigned with same value. Or I'm not understanding something here. :sweat_smile:

Reply:
There's nothing wrong here. The value should be updated on each boot (See FIXME).